CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+Releasing
+=========
+
+Summary
+-------
+
+- Create a release branch in the main repository, not in a fork.
+- Prepare a commit for a release candidate, get an approving review, and tag the commit which will upload artifacts to PyPI.
+- Notify people and give time for feedback.
+- Address concerns raised via new issues for later handling or PRs against the release branch.
+- Prepare a commit for a final release, get an approving review, and tag the commit which will upload artifacts to PyPI.
+
+
+Step-by-step
+------------
+
+.. note::
+
+    Commands are written with Linux in mind and a ``venv`` located in ``venv/``.
+    Adjust per your OS and virtual environment location.
+    For example, on Windows with an environment in the directory ``myenv/`` the Python command would be ``myenv/scripts/python``.
+
+- Define the final release version you are preparing.
+
+  - towncrier uses `CalVer <https://calver.org/>`_ of the form ``YY.MM.micro`` with the micro version just incrementing.
+  - Normalize the version according to `PEP 440 <https://www.python.org/dev/peps/pep-0440/#normalization>`_.
+
+    - This requires that ``towncrier[dev]`` extra is installed.
+    - ``venv/bin/python admin/canonicalize_version.py 19.09.00-rc1``
+    - Outputs ``19.9rc1`` which is the form to be used.
+
+- Create a release branch with a name of the form ``release-19.9`` starting from the ``master`` branch.
+
+  - This new branch will contain all tagged release candidate commits as well as the final tagged release commit.
+
+- Update the version to the release candidate with the first being ``rc1`` (as opposed to 0).
+
+  - In ``src/towncrier/_version.py`` the version is set using ``incremental`` such as ``__version__ = Version('towncrier', 19, 9, release_candidate=1)``
+
+- Run ``venv/bin/towncrier build --yes`` to build the the newsfragments into the release notes document and remove the newsfragment files.
+
+- Commit and push to the primary repository, not a fork.
+
+  - It is important to not use a fork so that pushed tags end up in the primary repository.
```

Comment:
I think that we can keep it simple. One reason is for tags, the other reason is that forks don't have access to PYPI secrets.


```suggestion
- Commit and push to the primary repository, not a fork.
```

Reply:
Explaining why is detrimental here?  hmm...

```suggestion
- Commit and push to the primary repository, not a fork.

  - It is important to not use a fork so that pushed tags end up in the primary repository, server provided secrets for publishing to PyPI are available, and maybe more.
```